docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -0,0 +1,88 @@
+# Mbed USB WAV Audio Player
+This is a tutorial on how to put together a USB WAV audio player with Mbed. In general, the user will load a WAV file onto an SD card that will be connected to an Mbed board. The user can then play the WAV file from the board on a host computer over USB. Once connected, the Mbed board will act as a USB microphone that will be playing the sound, so that the sound can be looped back to speakers, headphones, or recorded with a program like Audacity.
```

Comment:
Stating that the microphone will be playing sound ```Once connected, the Mbed board will act as a USB microphone that will be playing the sound``` is a little confusing. Explaining that it appears like a USB microphone but isn't actually recording anything might make this easier to understand. Maybe something like this:

```
Once connected, the Mbed board will act as a USB microphone. The sound that the microphone is "recording" is actually sound data read from the SD card and sent to the host computer. This allows the sound to be played on speakers, headphones, or recorded with a program like Audacity.
```